plugin/src/main/java/spp/jetbrains/sourcemarker/settings/LiveLogConfigurationPanel.java

```diff
@@ -49,15 +50,15 @@ public LiveLogConfigurationPanel(AutocompleteField autocompleteField, InlayMark
             try {
                 editorsProvider = (XDebuggerEditorsProvider) Class.forName(
                         "com.jetbrains.python.debugger.PyDebuggerEditorsProvider"
-                ).newInstance();
+                ).getDeclaredConstructor().newInstance();
```

Comment:
Please explain this change

Reply:
class.newInstance is deprecated and this is the alternate suggested.